.github/workflows/test_size.yml

```diff
@@ -111,12 +111,15 @@ jobs:
         shell: bash
         run: |
           python3 -m pip install -U weasyprint elf_diff anytree
-          python3 -m elf_diff --bin_prefix=arm-none-eabi- $GITHUB_WORKSPACE/master_bin/arduplane $GITHUB_WORKSPACE/pr_bin/arduplane
-          tar cf multipage_pair_report.tar multipage_pair_report
+          python3 -m elf_diff --bin_prefix=arm-none-eabi- --html_dir=plane $GITHUB_WORKSPACE/master_bin/arduplane $GITHUB_WORKSPACE/pr_bin/arduplane
+          python3 -m elf_diff --bin_prefix=arm-none-eabi- --html_dir=copter $GITHUB_WORKSPACE/master_bin/arducopter $GITHUB_WORKSPACE/pr_bin/arducopter
+          mkdir elf_diff
+          tar cf elf_diff/plane.tar plane
+          tar cf elf_diff/copter.tar copter
```

Comment:
Can we make this a loop, please?

Reply:
Sure I'll figure out how to do the loop here. And do it separately. 

The zip file thing I might leave for now .....